docs/.gitignore

```diff
@@ -1,2 +1,8 @@
-_site
+*.gem
+.bundle
+.ruby-version
+.jekyll-cache
 .sass-cache
+_site
+Gemfile.lock
```

Comment:
I don't think we should ignore this file, it's important for reproducible builds